k6POST.js

```diff
@@ -0,0 +1,41 @@
+import http from "k6/http";
+import { check, sleep } from "k6";
+
+export let options = {
+  stages: [
+    {duration: "30s", target: 150},
+    {duration: "270s", target: 150},
+  ]
+};
+
+// skew requests towards end of database per Learn Requirement
+let min = 8000000;
+let max = 10000000;
+const fakeText = ['dksalfjdasl ewqr d aic aewqr aka da', 'afkldfq caskl qeri das fc al', 'jdakl feqlk fda ca ewq', 'fjdkal eq pc qipurexnm'];
+const fakeDates = ['2019-01-01', '2018-10-10', '2019-04-04', '2019-05-01'];
+const fakeBooleans = ['t', 'f'];
```

Comment:
I have the same problem using faker in k6 script. I can only manually set fake data but using faker in this file. Let me know if you have figured out how to do it.

Reply:
For the fake data I don't think it really matters in this case right? The goal of the K6 stress testing is to test the latency of the POST request, not add real information to the database. That's why I just went with a few options. I delete them from my database each day after I run tests with a simple DELETE sql query.